src/status_im/chat/views/message/message.cljs

```diff
@@ -47,7 +47,7 @@
    incoming-name [:contact-name-by-identity from]]
   (if-let [name (if outgoing
                   (:name current-account)
-                  (or incoming-name "Unknown contact"))]
+                  (or incoming-name "Anonymous contact"))]
```

Comment:
Replace the hardcoded string with a call to `generate-gfy` - see comment on PR for the context

Reply:
Ok I understand know what "anonymous" actually means. Thanks a lot for the detailed explanation!